libs/application/templates/aosh/transfer-of-machine-ownership/src/fields/BuyerField/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { getErrorViaPath, getValueViaPath } from '@island.is/application/core'
```

Comment:
This does not need to be custom component. Better to extract the nationalIdWithName fields to a shared component. And for the email use a texfield with email variant and for the phone use the phoneField for consistancy accross all forms.